src/actions.tsx

```diff
@@ -0,0 +1,41 @@
+import * as constants from './constants';
+import { DataView } from './types';
+import { ConseilFilter } from './Conseil';
+
+export interface SwitchTab {
+    type: constants.SWITCH_TAB;
+    dataView: DataView;
+}
+
+export interface SetFilter {
+    type: constants.SET_FILTER;
+    filters: ConseilFilter;
+}
+
+export interface SetNetwork {
+    type: constants.SET_NETWORK;
+    network: string;
+}
+
+export type ArronaxAction = SwitchTab | SetFilter | SetNetwork;
+
+export function switchTab(dataView: DataView): SwitchTab {
+    return {
+        type: constants.SWITCH_TAB,
+        dataView: dataView
```

Comment:
You can do [object shorthand](https://github.com/airbnb/javascript#es6-object-concise) with es6, it's bomb 💣:

```
return {
  type: constants.SWITCH_TAB,
  dataView,
}
```

Reply:
Done!